airflow/models/baseoperator.py

```diff
@@ -1741,6 +1788,28 @@ def leaves(self) -> List["MappedOperator"]:
     def has_dag(self):
         return self.dag is not None
 
+    @property
+    def inherits_from_dummy_operator(self):
+        """Used to determine if an Operator is inherited from DummyOperator"""
+        # This looks like `isinstance(self, DummyOperator) would work, but this also
+        # needs to cope when `self` is a Serialized instance of a DummyOperator or one
+        # of its sub-classes (which don't inherit from anything but BaseOperator).
+        return getattr(self, '_is_dummy', False)
```

Comment:
`_is_dummy` has a default, so when can it be unset?

Reply:
Possibly can't be, I'll check.

Reply:
`SerializedBaseOperator` inherits from BaseOperator too so `_is_dummy` should be set